### PR TITLE
Set disabled flags on inputs

### PR DESCRIPTION
**Pull request by rhark1**
Current disabled functionality is not propagated to the actual UI components.  This prevents the UI components from displaying the correct disabled UI.  This PR just passes the disabled flag to the Keen UI components.

### CODE REVIEW

File: inputs/segmented-button.vue [hunk 1]
```diff
@@ -63,6 +64,8 @@
         const data = this.data,
           assetLocation = _.get(this.$store, 'state.site.assetHost') || _.get(this.$store, 'state.site.assetPath');
 
+        console.log('options', this.args.options);
```
Comment (jpope19) [inline inputs/segmented-button.vue]: remove log plz

File: inputs/wysiwyg.vue [hunk 2]
```diff
@@ -695,6 +695,10 @@
         }
       });
 
+      if (this.schema.disabled) {
```
Comment (MariaFdezJ) [inline inputs/wysiwyg.vue]: Why are we taking it from `schema` here, and not from `props` as the others?

File: inputs/simple-list-input.vue [hunk 3]
```diff
@@ -55,7 +57,7 @@
   import autocomplete from './autocomplete.vue';
 
   export default {
-    props: ['items', 'allowRepeatedItems', 'autocomplete', 'currentItem', 'disabled', 'isInitialFocus', 'ignoreComma'],
+    props: ['items', 'allowRepeatedItems', 'autocomplete', 'currentItem', 'disabled', 'isInitialFocus', 'ignoreComma', 'disabled'],
```
Comment (MariaFdezJ) [inline inputs/simple-list-input.vue]:
```suggestion
    props: ['items', 'allowRepeatedItems', 'autocomplete', 'currentItem', 'disabled', 'isInitialFocus', 'ignoreComma'],
```

It's already added, after`currentItem` and is adding `classes` for that. Is it actually needed for this component?

File: inputs/range.vue [hunk 4]
```diff
@@ -230,6 +230,10 @@
           el.noUiSlider.on(key, this.schema.events[key]);
         });
       }
+
+      if (this.disabled) {
+        el.setAttribute('disabled', this.disabled);
+      }
```
Comment (MariaFdezJ) [inline inputs/range.vue]: Can't we add it directly to this line: https://github.com/clay/clay-kiln/blob/2d5759f9ca024859363d3562052a505ccb8f5475/inputs/range.vue#L40 as `:disabled="disabled"` to keep consistency with the others?

File: inputs/wysiwyg.vue (hunk 2 above)
Comment (rhark1) [reply inputs/wysiwyg.vue]: I didn't want anyone to confuse it with the `isDisabled` data prop being used in the same component

File: inputs/range.vue (hunk 4 above)
Comment (rhark1) [reply inputs/range.vue]: The attribute is being set on the slider created in the `mounted` hook, not the actual element

**Pull request merged**